src/net/sourceforge/kolmafia/textui/Parser.java

```diff
@@ -2843,9 +2870,23 @@ private Evaluable parseCall(final BasicScope scope, final Evaluable firstParam)
     if (target != null) {
       params = this.autoCoerceParameters(target, params, scope);
     } else {
-      this.undefinedFunctionError(name, params);
-
       target = new BadFunction(name.content);
+
+      // Don't make an error if the function couldn't be found
+      // because the user messed up one of the parameters
+      // (since that means we already made one earlier)
+      boolean error = true;
```

Comment:
You could write this like

`boolean alreadyErrored = params.stream().anyMatch(param -> param != null && param.getType().isBad());`

Reply:
You could even do a parallel stream!